### DISCUSSION TITLE
Using hash_path seems to ignore slash_path_empty plugin.

### DISCUSSION THREAD

**Discussion by mmarvb8h** [jeremyevans/roda#321] [General]
Began using hash_paths and have noticed my caller cannot append a "/" to the end of the url. If so the slash_path assumes it is not a match. So i need:

    hash_path("api", "/auth") do |r|
    end

    hash_path("api", "/auth/") do |r|
    end

Using the slash_path_empty plugin fixes this using "standard" manual routing. Is this expected behavior?

thx ~Marvin

**Comment by jeremyevans** [comment]
Yes, this is expected.  For hash_paths, the path must match exactly, because it is used to reference into a hash, not via normal routing.  It wouldn't be possible to get this to work by default unless one of the plugins depended on either other, which is undesirable.  It's possible we could allow for the behavior via a new plugin.

**Reply by marvfinsy** [reply]
cool... will try hash_branches and use "r.is do...end" thx...
